src/net/tcp/listener.rs

```diff
@@ -25,18 +29,25 @@ impl TcpListener {
     ///
     /// The returned listener is ready for accepting connections.
     ///
-    /// Only 0.0.0.0 is currently supported.
+    /// If you bind to the 0.0.0.0, you're effectivly binding to the generated IP address of the host.
+    /// See the Dns class for the current IP assignment.
```

Comment:
Rust doesn't have classes and this is public documentation, which doesn't know about `Dns`.